tests/src/rules/no-default-export.js

```diff
@@ -0,0 +1,129 @@
+import { test } from '../utils'
+
+import { RuleTester } from 'eslint'
+
+const ruleTester = new RuleTester()
+    , rule = require('rules/no-default-export')
+
+ruleTester.run('no-default-export', rule, {
+  valid: [
+    test({
+      code: `
+        export const foo = 'foo';
+        export const bar = 'bar';`,
+      }),
+    test({
+      code: `
+        export const foo = 'foo';
+        export function bar() {};`,
+      }),
+    test({
+      code: `
+        export const foo = 'foo';`,
+      }),
+    test({
+      code: `
+        const foo = 'foo';
+        export { foo };`,
+      }),
+    test({
+      code: `
+        export { foo, bar }`,
+      }),
+    test({
+      code: `
+        export const { foo, bar } = item;`,
+      }),
+    test({
+      code: `
+        export const { foo, bar: baz } = item;`,
+      }),
+    test({
+      code: `
+        export const { foo: { bar, baz } } = item;`,
+      }),
+    test({
+      code: `
+        export const foo = item;
+        export { item };`,
+      }),
+    test({
+      code: `
+        export * from './foo';`,
+      }),
+    test({
+      code: `
+        export const { foo } = { foo: "bar" };`,
+      }),
+    test({
+      code: `
+        export const { foo: { bar } } = { foo: { bar: "baz" } };`,
+      }),
+    test({
+      code: 'export { a, b } from "foo.js"',
+      parser: 'babel-eslint',
+      }),
```

Comment:
all these `}),` need to be de-indented a level (it seems like this is already the way it is in the prefer-default-export tests, but let's fix it here anyways)

Reply:
the indentation here is still off; the `}` should match the indentation level of the line that ends with `}`